plugins/delete/delete.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const path = require('path');
+const Plugins = require('../../modules/Plugins.js');
+
+module.exports = {
+  name: 'Delete',
+  description: 'delete commands',
+  chat: true, // Defines this as a Chat Command
+  event: false, // Is this a Event?
+  type: 5004, // Type Event
+  command: 'delete', // This is the Command that is typed into Chat!
+  permissions: ['moderator', 'creator'], // This is for Permissisons depending on the Platform.
+  alias: [], // Alias commands that preform interesting things.
+  cooldown: 1, // this is Set in Seconds, how long between the next usage of this command.
+  settings: false, // Defining this as false will load the Settings file for this Plugin when the system loads this plugin.
+  credits: 'Created by Rehkloos',
+  execute(client, data) {
+    const command = data.args[0];
+    const dir = path.join(__dirname, '..');
+    const fileName = `${command}.js`;
+    if (command.includes('.')) {
+      client.sendMessage('Something is not fuctioning correctly');
+      return;
+    }
+    if (fs.existsSync(dir)) {
+      fs.rmdirSync(path.join(dir, command), {
+        // check directory and args
+        recursive: true, // delete parameters recursively
+      });
+
+      const newPluginDir = path.join(__dirname, `./../`, command, `/`, fileName);
```

Comment:
You did it in wrong order. You should first delete it from Plugins.chat and then delete folder. Dont know if you can require a deleted file. :D